.github/workflows/update_robottelo_image.yml

```diff
@@ -0,0 +1,97 @@
+name: update_robottelo_image
+
+on: [push]
+
+jobs:
+  codechecks:
+    name: Code Quality
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8, 3.9]
+    env:
+      PYCURL_SSL_LIBRARY: gnutls
+    steps:
+      - name: Checkout Robottelo
+        uses: actions/checkout@v2
+
+      - name: Set Up Python-${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install Dependencies
+        run: |
+          sudo apt-get install -y libgnutls28-dev libcurl4-openssl-dev libssl-dev
+          wget https://raw.githubusercontent.com/SatelliteQE/broker/master/broker_settings.yaml.example
+          pip install -U --no-binary=pycurl -r requirements.txt -r requirements-optional.txt
+          cp robottelo.properties.sample robottelo.properties
+          cp broker_settings.yaml.example broker_settings.yaml
+          cp virtwho.properties.sample virtwho.properties
+
+      - name: Pre Commit Checks
+        uses: pre-commit/action@v2.0.0
+
+      - name: Collect Tests
+        run: |
+          pytest -n 8 --setup-plan --disable-pytest-warnings tests/foreman/ tests/robottelo/
+          pytest -n 8 --setup-plan --disable-pytest-warnings -m pre_upgrade tests/upgrades/
+          pytest -n 8 --setup-plan --disable-pytest-warnings -m post_upgrade tests/upgrades/
+
+      - name: Test Robottelo Coverage
```

Comment:
Do we really need this check for pushing the robottelo image to Quay ?

The code-coverage shouldnt have anything to do with image , I would remove it.

Reply:
@jyejare probably we don't need it. It's just that I have separated the workflow which we currently have, that's why you are seeing this here. If you want I can remove this from here but then should I keep the push event which I removed? 
Currently the main robottelo_container job depends on codechecks job, it's upto us if we want it this way.

@JacobCallahan any thoughts?

Reply:
I agree. this is something I was initially thinking as well.

Reply:
removed